sdk/authorization/arm-authorization/review/arm-authorization.api.md

```diff
@@ -8,1289 +8,233 @@ import * as coreAuth from '@azure/core-auth';
 import * as coreClient from '@azure/core-client';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 
-// @public
-export type ApprovalMode = string;
-
```

Comment:
@qiaozha Could you also help review this pr?

Since there are a lot of breaking changes since last relased.